RuleDocumentation/AvoidUsingDoubleQuotesForConstantString.md

```diff
@@ -4,18 +4,27 @@
 
 ## Description
 
-When the value of a string is constant (i.e. not being interpolated by injecting variables or expression into such as e.g. `"$PID-$(hostname)"`), then single quotes should be used to express the constant nature of the string. This is not only to make the intent clearer that the string is a constant and makes it easier to use some special characters such as e.g. `$` within that string expression without the need to escape them. There are exceptions to that when double quoted strings are more readable though, e.g. when the string value itself has to contain a single quote (which would require a double single quotes to escape the character itself) or certain very special characters such as e.g. `"\n"` are already being escaped, the rule would not warn in this case as it is up to the author to decide on what is more readable in those cases.
+Single quotes should be used when the value of a string is constant. A constant string doesn't
+contain variables or expressions intended to insert values into the string, such as
+`"$PID-$(hostname)"`).
+
+This makes the intent clearer that the string is a constant and makes it easier to use some special
+characters such as `$` within that string expression without needing to escape them.
+
+There are exceptions to that when double quoted strings are more readable. For example, when the
+string value itself must contain a single quote or other special characters (such as `"``n"`) are
```

Comment:
does the example need to be `"\n"` instead of `"``n"`?

Reply:
In PowerShell `` `n `` is the escape sequence for the newline character.